graylog2-web-interface/src/components/search/SidebarMessageField.jsx

```diff
@@ -1,12 +1,11 @@
 import React, {PropTypes} from 'react';
-import {Input, Button, ButtonGroup} from 'react-bootstrap';
+import {Input, Button, ButtonGroup, DropdownButton, MenuItem} from 'react-bootstrap';
```

Comment:
Button and ButtonGroup are not used any more, we can get rid of them and the lint error.